sdk/attestation/Azure.Security.Attestation/README.md

```diff
@@ -219,13 +220,19 @@ There are two properties provided in the [PolicyResult][attestation_policy_resul
 To verify the hash, clients can generate an attestation token and verify the hash generated from that token:
 
 ```C# Snippet:VerifySigningHash
-// The SetPolicyAsync API will create a SecuredAttestationToken to transmit the policy.
-var policySetToken = new SecuredAttestationToken(new StoredAttestationPolicy { AttestationPolicy = attestationPolicy }, TestEnvironment.PolicySigningKey0, policyTokenSigner);
-
-var shaHasher = SHA256Managed.Create();
+// The SetPolicyAsync API will create an AttestationToken signed with the TokenSigningKey to transmit the policy.
+// To verify that the policy specified by the caller was received by the service inside the enclave, we
+// verify that the hash of the policy document returned from the Attestation Service matches the hash
+// of an attestation token created locally.
+TokenSigningKey signingKey = new TokenSigningKey(<Customer provided signing key>, <Customer provided certificate>)
+var policySetToken = new AttestationToken(
+    new StoredAttestationPolicy { AttestationPolicy = attestationPolicy },
+    signingKey);
+
+using var shaHasher = SHA256Managed.Create();
 var attestationPolicyHash = shaHasher.ComputeHash(Encoding.UTF8.GetBytes(policySetToken.ToString()));
 
-CollectionAssert.AreEqual(attestationPolicyHash, setResult.Value.PolicyTokenHash);
+Debug.Assert(attestationPolicyHash.SequenceEqual(setResult.Value.PolicyTokenHash));
```

Comment:
Nice!  I would not have thought of an assert here, but it fits well.

Reply:
For clients that need to validate that the enclave received the policy, this sample is supposed to indicate best practices for validation indication that the client can/should verify that the hash matches.